x-pack/plugins/logstash/public/plugin.ts

```diff
@@ -30,6 +30,7 @@ interface SetupDeps {
 
 export class LogstashPlugin implements Plugin<void, void, SetupDeps> {
   private licenseSubscription?: Subscription;
+  private capabilities$ = new BehaviorSubject<null | Capabilities>(null);
```

Comment:
__nit:__ it feels like you can just use `new Subject<Capabilities>()` and remove null check in the subscription?

Reply:
Good call, will do!